invenio/ext/login/legacy_user.py

```diff
@@ -59,6 +67,20 @@
 }
 
 
+def isUserAdmin(user_info):
+    """Return True if the user has some admin rights; False otherwise."""
+    user = User.query.get(user_info['uid'])
+    return user and user.has_admin_role
+
+def isUserSuperAdmin(user_info):
+    """Return True if the user is superadmin; False otherwise."""
+    user = User.query.get(user_info['uid'])
+    if user and user.has_super_admin_role:
+        return True
+    return acc_firerole_check_user(
```

Comment:
@jirikuncar do you think we need to move this part directly inside the [has_super_admin_role](https://github.com/inveniosoftware/invenio/blob/master/invenio/modules/access/models.py#L255) property?